app/controllers/mixins/actions/vm_actions/resize.rb

```diff
@@ -33,14 +43,18 @@ def resize
             )
           end
           @sb[:explorer] = @explorer
+          @request_id = params[:req_id] ? params[:req_id] : nil
```

Comment:
this is just `@request_id = params[:req_id]`

Reply:
fixing this...